libs/wingsdk/src/std/array.ts

```diff
@@ -42,7 +42,7 @@ export class Array {
   /**
    * Get the value at the given index, returning nil if the index is out of bounds.
    *
-   * @macro $self$.at($args$)
+   * @macro $self$?.at($args$)
```

Comment:
I'm not sure if this is needed
```suggestion
   * @macro $self$.at($args$)
```